tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -449,6 +450,70 @@ def teardown():
+                    sc_obj = storageclass_factory(
+                        interface=constants.CEPHFILESYSTEM,
+                        sc_name=cephfs_sc_name,
+                        replica=replica,
+                        compression=compression,
+                        erasure_coded=erasure_coded,
+                        new_cephfs_pool=True,  # True because the requirement of new SC in DR is with non default pool
+                    )
+                    if sc_obj is None or sc_obj.name != cephfs_sc_name:
+                        log.error(
+                            f"Failed to create SC '{cephfs_sc_name}' or name mismatch: "
+                            f"Created '{sc_obj.name if sc_obj else 'None'}'"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)